pom.xml

```diff
@@ -166,6 +167,19 @@
   </properties>
 
   <repositories>
+    <repository>
+      <id>avro-util-repository</id>
+      <name>datanucleus maven repository</name>
```

Comment:
Update the name. This seems to be a copy-paste of the item below.

Reply:
Changed to Avro util repo name.